client/src/views/NABat/NABatSpectrogram.vue

```diff
@@ -87,15 +88,29 @@ export default defineComponent({
       const response = compressed.value
         ? await getNABatSpectrogramCompressed(props.id, props.apiToken)
         : await getNABatSpectrogram(props.id, props.apiToken);
-      if (response.data["url"]) {
-        image.value.src = response.data['url'];
+      if (response.data.urls.length) {
+        const urls = response.data.urls;
```

Comment:
Does this need to check to see if `data.urls` is an array or a string? If this is getting merged into `main` before the server changes, will this break?

Reply:
The server has implemented a change so that any access to the spectrogram or compressed spectrogram for both NABat and BatAI will return a list of one.

I know this is named `client-url-list` but I did change stuff in the ./views so that it will returna  list of a single image.

Reply:
Indeed, you did